robottelo/common/constants.py

```diff
@@ -144,6 +144,7 @@
 REPOSET = {
     'rhct6': "Red Hat CloudForms Tools for RHEL 6 (RPMs)",
     'rhel6': "Red Hat Enterprise Linux 6 Server (RPMs)",
+    'rhelc6': "Red Hat Enterprise Linux 6 Server - RH Common (RPMs)",
```

Comment:
Eventually we will need to start using ``Red Hat Satellite Tools 6 Beta (for RHEL 6 Server) (RPMs)`` as the **RH Common** channel will become 'obsolete'